.gitignore

```diff
@@ -2,3 +2,5 @@ target/
 output/
 .idea/
 .vscode/
+tracing.folded
```

Comment:
Since the `--profile` now takes in a path, this should not be necessary

Reply:
I'm confused about how to pass the path in

i run --profile tempfile
and i get
error: profile 'tempfile' is not defined

what am i doing wrong?

Reply:
Try `cargo run -- --profile <path> deploy examples/fibonacci-c:0.3.0`